decidim-core/app/models/decidim/user.rb

```diff
@@ -214,7 +214,7 @@ def interested_scopes
 
     # Caches a Decidim::DataPortabilityUploader with the retrieved file.
     def data_portability_file(filename)
-      @data_portability_file ||= DataPortabilityUploader.new.tap do |uploader|
+      @data_portability_file ||= DataPortabilityUploader.new(user).tap do |uploader|
```

Comment:
I'm not sure about this line, should it be this?

```suggestion
      @data_portability_file ||= DataPortabilityUploader.new(self).tap do |uploader|
```

Reply:
Both are working. As this is a backport I suggest being consistent and use the same parameters.

Reply:
Agree, thanks for taking the time to check it out!